playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -436,3 +437,56 @@ def foreign_logs_enricher(event: ExecutionBaseEvent, params: ForeignLogParams):
 
 
 logs_enricher = action(logs_enricher)
+
+
+class MentionParams(ActionParams):
+    static_mentions: Optional[List[str]]
```

Comment:
I would add a parameters docs section, which would be rendered in the docs:
In the slack id example, I would put the format of real Slack ids
```
    """
    :var static_mentions: List of Slack user ids/groups ids to be mentioned
    :var mentions_label: A alert label, or Kubernetes resource label, in which the value contains a comma separated ids to mention
    :var message_template: Optional. Custom mention message. Default: `"Hey: $mentions"`

    :example static_mentions: ["slack id1", "slack id2"]
    """

```